Resources/Prototypes/_Goobstation/Roles/Jobs/Command/blueshield_officer.yml

```
@@ -58,6 +58,7 @@
     belt: BlueShieldShield
   storage:
     back:
+    - ClothingMaskGasSecurity 
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)